### ISSUE TITLE
Laravel 5.2

### ISSUE THREAD

**Issue by heikobihlmaier** [pingpong-labs/shortcode#9]
Laravel 5.2 Support is missing.

@gravitano Can u implement this ?

**Comment by adrianthedev** [comment]
+1

Thank you!

**Comment by heikobihlmaier** [comment]
@gravitano  Status ???

**Comment by gravitano** [comment]
@heikobihlmaier Do you get any errors? This package is not require any `illuminate` package.

**Comment by heikobihlmaier** [comment]
With version tag 2.2 I get an error, because thunder/shortcode is not available in any stable version. I need to add 

```
"minimum-stability": "dev",
"prefer-stable":     true
```

for install.

Without that the installation of your package is not possible.
Addional is your documenation of this package not useful, because there is nothing correct.

I used it so: is this correct ?

```
\Shortcode::register('icon', function(ShortcodeInterface $shortcode) {
            echo $shortcode->getName().': '.$shortcode->getContent() . ':' . $shortcode->getParameter('lib');
        });
```
Greetings from Germany.

**Comment by gravitano** [comment]
Try install the latest commit from branch `2.2`. I've been tried in my Local machine and everything is fine.

**Comment by thunderer** [comment]
@heikobihlmaier Yes, `thunderer/shortcode` was in `dev-master` here, but it is changed to `0.5.*` in branch `2.2`. I suggest changing it to `^0.5` so that it'll automatically cover new releases in `0.*`. @gravitano Could you please release `2.2.0` with said changes? If there are any questions, let me know how could I help.
